en/docs/deploy/aws.md

```diff
@@ -0,0 +1,88 @@
+
+
+    6.  Add the WSO2 Helm chart repository.
+        ```curl                                                                
+        helm repo add wso2 https://helm.wso2.com && helm repo update
+        ```
+
+!!! note ""
+	-	The local copy of the `wso2/kubernetes-is` git repository will be referred to as `KUBERNETES_HOME`.
+	-	`<KUBERNETES_HOME>/advanced/` will be referred to as `HELM_HOME`.
+
+### Step 1 - Clone the Kubernetes resources from the WSO2 Identity Server git repository
+
+```java
+git clone https://github.com/wso2/kubernetes-is.git
+```
+
+### Step 2 - Change the configurations as required 
+
+1.	The default configurations work well for the basic deployment of the product. However, if there is anything specific that needs to be configured, change the respective files in `<HELM_HOME>/is-pattern-1/`. 
+
+2.	 Open `<HELM_HOME>/is-pattern-1/values.yaml` and provide the values as mentioned in the second step **(Provide configurations)** of the **Quick Start Guide** [here](https://hub.helm.sh/charts/wso2/is-pattern-1).
+
+### Step 3 - Deploy WSO2 Identity Server
+
+Execute the following command to deploy the product. Here, `NAMESPACE` is the Kubernetes Namespace in which the resources are deployed and the `<RELEASE_NAME>` can be any name that you choose for the deployed instance. 
+
+```java
+helm install --dep-up --name <RELEASE_NAME> <HELM_HOME>/is-pattern-1 --namespace <NAMESPACE>
+```
+
+### Step 4 - Access the management console
+
+To access the console in the environment,
+
+1.	Obtain the external IP of the Ingress resources by listing down the Kubernetes Ingresses as shown below. 
+
+	```java
+	kubectl get ing -n <NAMESPACE>
+	```
+	This gives you the `<RELEASE_NAME>` and `<EXTERNAL-IP>` as shown below. 
+
+	```java 
+	NAME                       HOSTS                  ADDRESS        PORTS     AGE
+	wso2is-ingress             <RELEASE_NAME>         <EXTERNAL-IP>  80, 443   3m
+	```
+
+2.	Add the information obtained above in the /etc/hosts file as an entry. 
+
+	```java
+	<EXTERNAL-IP>	<RELEASE_NAME>
+	```
+
+3.	Navigate to `https://<RELEASE_NAME>/carbon` on a new browser window.
+
+### Try it Out 
+
+You can now test the functionalities of WSO2 Identity Server with your app. You can alternatively choose a sample app from [here](../../samples/overview) and follow the steps given to deploy the chosen application. 
+
+Make sure you add the proxy port configuration to `<KUBERNETES_HOME>/advanced/is-pattern-1/values.yaml`. 
+
+```toml
+[transport.http.properties]
+proxyPort = 80
+[transport.https.properties]
+proxyPort = 443
+```
+
+!!! important 
+	The host name has to be changed in the URLs related to the identity provider based on the `<RELEASE_NAME>` you chose in [step-3](#step-3-deploy-wso2-identity-server). This configuration is present in the properties file in `<SAMPLE_HOME>/WEB-INF/classes` where`<SAMPLE_HOME>` refers to the sample application that you have chosen to verify this deployment. 
```

Comment:
The host name included in the URLs related to the identity provider must be changed based on the `<RELEASE_NAME>`......